lib/onc_certification_g10_test_kit/bulk_data_group_export_stu1.rb

```diff
@@ -244,7 +244,8 @@ class BulkDataGroupExportSTU1 < Inferno::TestGroup
 
         Each file SHALL contain resources of only one type, but a server MAY create more than one file for each resource type returned.
 
-        * url - the path to the file. The format of the file SHOULD reflect that requested in the _outputFormat parameter of the initial kick-off request.
+        * url - the #{'absolute' if config.options[:require_absolute_urls_in_output]} path to the file.
```

Comment:
```suggestion
        * url - the #{'absolute' if config.options[:require_absolute_urls_in_output]} path to the file.
```
This string interpolation isn't working as intended -- seems _config.options_ is an empty hash at the time of interpolation. If there's a quick alternative, let me know.

Reply:
Descriptions are evaluated at boot time, and outside of the context of a test session, so suite options are unavailable. Hacking in a workaround doesn't seem worth it for such a minor change.